source/users/create.txt

```diff
@@ -226,39 +345,67 @@ There is no such specific method to re-run a custom confirmation
 function. Instead, App Services has a method to trigger a re-run of whatever
 the current Email/Password user confirmation workflow happens to be. You
 can manually re-run the currently selected user confirmation flow using
-the :doc:`App Services Admin API </admin/api/v3>` or via the App Services UI:
+the the App Services UI or the :doc:`App Services Admin API </admin/api/v3>`:
 
-To re-run the confirmation workflow for a user through the App Services UI:
+.. tabs-realm-admin-interfaces::
+   
+   .. tab::
+      :tabid: ui
 
-#. Select :guilabel:`App Users` from the left navigation menu.
+      To re-run the confirmation workflow for a user in the UI:
 
-#. Under the :guilabel:`Users` tab, select the :guilabel:`PENDING`
-   button.
+      #. Select :guilabel:`App Users` from the left navigation menu.
 
-#. Find the user in the list and click on the ellipsis (``...``).
+      #. Under the :guilabel:`Users` tab, select the :guilabel:`PENDING`
+         button.
 
-#. Select the :guilabel:`Run user confirmation` option from the context
-   menu that appears.
+      #. Find the user in the list and click on the ellipsis (``...``).
 
-#. Select the :guilabel:`Run User Confirmation` button in the dialogue
-   box that appears.
+      #. Select the :guilabel:`Run user confirmation` option from the context
+         menu that appears.
 
-#. If the operation succeeds, the banner at the top of the App Services
-   admin console should display a confirmation message. The user's
-   :guilabel:`User Status` changes from :guilabel:`Pending Confirmation`
-   to :guilabel:`Pending User Login`. Once the user logs in, they will
-   move into the active users list automatically.
+      #. Select the :guilabel:`Run User Confirmation` button in the dialogue
+         box that appears.
 
-If the re-run fails or the user's tokens expire again, you can re-run
-the confirmation function as many times as necessary.
+      #. If the operation succeeds, the banner at the top of the App Services
+         admin console should display a confirmation message. The user's
+         :guilabel:`User Status` changes from :guilabel:`Pending Confirmation`
+         to :guilabel:`Pending User Login`. Once the user logs in, they will
+         move into the active users list automatically.
```

Comment:
```suggestion
         move into the active user's list automatically.
```

Does the active user owner the list? If so, it should be 'user's list'

Reply:
The user doesn't own the list, but the wording is a bit confusing. I'll change `active users list` to `list of active users`.

Reply:
Oh language. Thanks for the fix